care/emr/migrations/0010_facilitylocation_facilitylocationorganization.py

```
@@ -0,0 +1,71 @@
+                ('modified_date', models.DateTimeField(auto_now=True, db_index=True, null=True)),
+                ('deleted', models.BooleanField(db_index=True, default=False)),
+                ('history', models.JSONField(default=dict)),
+                ('meta', models.JSONField(default=dict)),
+                ('status', models.CharField(max_length=255)),
+                ('operational_status', models.CharField(max_length=255)),
+                ('name', models.CharField(max_length=255)),
+                ('description', models.CharField(max_length=255)),
+                ('mode', models.CharField(max_length=255)),
+                ('location_type', models.CharField(max_length=255)),
+                ('form', models.CharField(max_length=255)),
+                ('facility_organization_cache', django.contrib.postgres.fields.ArrayField(base_field=models.IntegerField(), default=list, size=None)),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)